api/tinygo/runnable/internal/ffi/result.go

```diff
@@ -0,0 +1,34 @@
+//go:build tinygo.wasm
+
+package ffi
+
+// #include <reactr.h>
+import "C"
+import (
+	"github.com/suborbital/reactr/api/tinygo/runnable/runnable"
+)
+
+func result(size int32) ([]byte, runnable.HostErr) {
+	allocSize := size
+
+	if size < 0 {
+		if size == -1 {
+			return nil, runnable.NewHostError("unknown error returned from host")
```

Comment:
Do the other languages return here? I'm wondering because not calling get_ffi_result can cause the host to retain that data in the buffer and can cause issues for subsequent host calls. Can't remember if a `-1` error from the host means it never filled the buffer at all.